### PR TITLE
feat(ur-sdk): add command parser

### PR DESCRIPTION
**Pull request by marktoda**
- feat: add command parser
- feat: start testing command parser

## PR Scope

Please title your PR according to the following types and scopes following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/):

- `fix(SDK name):` will trigger a patch version
- `chore(<type>):` will not trigger any release and should be used for internal repo changes
- `<type>(public):` will trigger a patch version for non-code changes (e.g. README changes)
- `feat(SDK name):` will trigger a minor version
- `feat(breaking):` will trigger a major version for a breaking change

## Description

_[Summary of the change, motivation, and context]_

## How Has This Been Tested?

_[e.g. Manually, E2E tests, unit tests, Storybook]_

## Are there any breaking changes?

_[e.g. Type definitions, API definitions]_

If there are breaking changes, please ensure you bump the major version Bump the major version (by using the title `feat(breaking): ...`), post a notice in #eng-sdks, and explicitly notify all Uniswap Labs consumers of the SDK.

## (Optional) Feedback Focus

_[Specific parts of this PR you'd like feedback on, or that reviewers should pay closer attention to]_

## (Optional) Follow Ups

_[Things that weren't addressed in this PR, ways you plan to build on this work, or other ways this work could be extended]_

### CODE REVIEW

**Comment by marktoda** [comment]
TODO: more tests, subparsers for v4 stuff

**Review by snreynolds — APPROVED** [review]
this looks good but I'm going to have @hensha256 take a look since she owned a lot of the other routing code, and reviewed other ur-sdk work!

File: sdks/universal-router-sdk/src/utils/routerCommands.ts [hunk 1]
```diff
@@ -42,35 +53,99 @@ const PERMIT_BATCH_STRUCT =
...
   // Token Actions and Checks
-  [CommandType.WRAP_ETH]: ['address', 'uint256'],
-  [CommandType.UNWRAP_WETH]: ['address', 'uint256'],
-  [CommandType.SWEEP]: ['address', 'address', 'uint256'],
-  [CommandType.TRANSFER]: ['address', 'address', 'uint256'],
-  [CommandType.PAY_PORTION]: ['address', 'address', 'uint256'],
-  [CommandType.BALANCE_CHECK_ERC20]: ['address', 'address', 'uint256'],
+  [CommandType.WRAP_ETH]: [
+    { name: 'recipient', type: 'address' },
+    { name: 'amountMin', type: 'uint256' },
```
Comment (hensha256) [inline sdks/universal-router-sdk/src/utils/routerCommands.ts:118]: i know its named amountMin in the contract but thats a mistake 😆 
its actually just `amount`

Comment (hensha256) [reply sdks/universal-router-sdk/src/utils/routerCommands.ts:118]: it goes into this function which is just amount
https://github.com/Uniswap/universal-router/blob/f95928955d26ec2b6a74058c63fa8c2747c734f6/contracts/modules/Payments.sol#L111

Comment (marktoda) [reply sdks/universal-router-sdk/src/utils/routerCommands.ts]: ty fixed!

**Review by hensha256 — APPROVED** [review]

**Pull request merged**